ext/shmop/shmop.stub.php

```diff
@@ -4,15 +4,15 @@
 
 final class Shmop {}
 
-function shmop_open(int $key, string $flags, int $mode, int $size): Shmop|false {}
+function shmop_open(int $key, string $mode, int $permissions, int $size): Shmop|false {}
 
-function shmop_read(Shmop $shmid, int $start, int $count): string {}
+function shmop_read(Shmop $shmop, int $offset, int $size): string {}
 
 /** @deprecated */
-function shmop_close(Shmop $shmid): void {}
+function shmop_close(Shmop $shmop): void {}
 
-function shmop_size(Shmop $shmid): int {}
+function shmop_size(Shmop $shmop): int {}
 
-function shmop_write(Shmop $shmid, string $data, int $offset): int {}
+function shmop_write(Shmop $shmop, string $content, int $offset): int {}
```

Comment:
I followed `file_put_contents()` here, although, I would be ok with `$data` too. `fwrite()` uses `$string`, and I think that should also be changed to either `$data` or `$content`. Or for anything we choose :)

Reply:
As we used `$data` for ext/session, let's go with it here as well.